Program.cs

```diff
@@ -124,22 +121,28 @@ static void MenuOptions(IOptions<SecretsModel> options)
             }
         }
 
-        static void WeatherForecast()
+        static void WeatherForecast(IOptions<SecretsModel> options)
         {
-            if (APIKEY.Length == 0) { Console.WriteLine("Whoops! API key is not defined."); return; }
+            var openWeatherMapApiKey = options.Value.OpenWeatherMapApiKey;
+            
+            if (String.IsNullOrEmpty(openWeatherMapApiKey)) {
+                 Console.WriteLine("Whoops! API key is not defined."); 
+                 return;
+            }
+
             Console.Write("Enter your town name:");
             string town = Console.ReadLine();
 
             string resp = "";
 
             using (var wc = new WebClient())
             {
-                resp = wc.DownloadString($"http://api.openweathermap.org/data/2.5/weather?q={town}&appid={APIKEY}");
+                resp = wc.DownloadString($"http://api.openweathermap.org/data/2.5/weather?q={town}&appid={openWeatherMapApiKey}");
             }
             WeatherRoot wr = JsonSerializer.Deserialize<WeatherRoot>(resp);
 
             Console.WriteLine();
-            Console.WriteLine("Temperature: " + Weather.KtoF(wr.main.temp) + "�F or " + Weather.KtoC(wr.main.temp) + "�C. Feels like: " + Weather.KtoF(wr.main.temp) + "�F or " + Weather.KtoC(wr.main.temp) + "�C");
+            Console.WriteLine("Temperature: " + Weather.KtoF(wr.main.temp) + "�F or " + Weather.KtoC(wr.main.temp) + "�C. Feels like: " + Weather.KtoF(wr.main.temp) + "�F or " + Weather.KtoC(wr.main.temp) + "�C");
```

Comment:
This line seems to contain corrupted characters. Perhaps, the files were saved in ASCII/deprecated encoding?

Reply:
@kolya5544 
Good catch!Thank you!

I have already fix this in my further commits.
State of the file in last commit (9debcfd) seems correct.